### DISCUSSION TITLE
Catalogos

### DISCUSSION THREAD

**Discussion by lgarcia1973** [SAT-CFDI/python-satcfdi#28] [Q&A]
Como funcionan los catalogos? 
vi las tablas en el archivo pero al utilizar el select importado de catalogos no obtengo informacion
basicmente buscando codigos postales en la tabla C756_c_CodigoPostal todos los que probe salen None

from satcfdi.catalogs import *
r = select('C756_c_CodigoPostal',85000)
print(r)

tambien probe con el codigo en texto

**Comment by satcfdi** [comment]
c.execute(f"SELECT value FROM {catalog_name} WHERE key = ?", (pickle.dumps(key),))
    if ds := c.fetchone():
        return pickle.loads(ds[0])

**Reply by satcfdi** [reply]
las llaves estan en binario

**Reply by lgarcia1973** [reply]
Gracias!
Olvide agradecer en su momento, nunca es tarde
